app/components/wallet/WalletTrezorDialog.js

```diff
@@ -628,21 +643,19 @@ export default class WalletTrezorDialog extends Component<Props, State> {
       const trezorEventDevice = { ...this.trezorEventDevice };
       const trezorValidity = this._validateTrezor(cardanoGetPublicKeyResp, trezorEventDevice);
 
-      const trezorDeviceInfo = {};
-      trezorDeviceInfo.valid = trezorValidity.valid;
-      trezorDeviceInfo.error = trezorValidity.error;
-      trezorDeviceInfo.cardanoGetPublicKeyResult = cardanoGetPublicKeyResp;
-      if (trezorEventDevice.payload.type === 'acquired') {
-        trezorDeviceInfo.features = trezorEventDevice.payload.features;
-      }
-      this.trezorDeviceInfo = trezorDeviceInfo;
-
+      const trezorDeviceInfo = Object.assign({}, trezorValidity);
```

Comment:
maybe we could use the other way to create an Object, like you did 2 lines above: `{ ...trezorValidity };`